### PR TITLE
Fixed broken macros for nozzle and status led on/off. 

### PR DESCRIPTION
**Pull request by claudioguareschi**
Still need to fix SET_LED macro to disable effects if leds are set when effects are running

### CODE REVIEW

File: install.sh
Comment (Frix-x) [inline install.sh:1]: Can you reset this file to the original for the PR?

File: moonraker/base.conf
Comment (Frix-x) [inline moonraker/base.conf:1]: Same here: can you reset this file to the original for the PR?

**Review by Frix-x — CHANGES_REQUESTED** [review]
Thanks for the fix and for following up on this issue. Your contributions are greatly appreciated :)

I just added a small change request to be able to merge the PR cleanly as everything else looks good to me.

File: install.sh
Comment (claudioguareschi) [reply install.sh:1]: Done

File: moonraker/base.conf
Comment (claudioguareschi) [reply moonraker/base.conf:1]: done

File: install.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ BACKUP_PATH="${HOME}/klippain_config_backups"
 # Where the Klipper folder is located (ie. the internal Klipper firmware machinery)
 KLIPPER_PATH="${HOME}/klipper"
 # Branch from Frix-x/klippain repo to use during install (default: main)
-FRIX_BRANCH="main"
+FRIX_BRANCH="develop"
```
Comment (Frix-x) [inline install.sh:30]: Sorry, you just miss this last line and it will be good to go ^^

Comment (claudioguareschi) [reply install.sh:30]: done

**Pull request merged**